src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -263,6 +270,7 @@ private void seekToEnd(RoutingContext routingContext) {
      * @param routingContext RoutingContext instance
      */
     private void processConsumer(RoutingContext routingContext) {
+        log.info("OpenAPI operation = {}", this.httpBridgeContext.getOpenApiOperation());
```

Comment:
You're relying here on getting the operation that the caller set on the context. But why not just do the logging in the caller when you're setting the operation on the context? That the setting and logging is done in the caller just like the code in the other handlers like `deleteConsumer()` and you can factor out a method to do the setting and logging. Or just do the logging in the `context.setOpenApiOperation`.